src/org/usfirst/frc/team1492/robot/Robot.java

```diff
@@ -127,6 +130,12 @@ public void teleopPeriodic() {
         driveBase.useHighGear(driveHighGear);
 
         winch.moveWinch(manipulator.getButton(Button.X));
+        
+        outfeed.moveOutfeed(manipulator.getAxis(Axis.RIGHT_Y));
+
+        if (winchKill.get()) {
+            winch.moveWinch(false);
+        }
```

Comment:
Remove this winch stuff, since we do not have a switch to stop the winch wired currently, and it is beyond the scope of this pull request..